Update Spring Cloud Azure README file (#37772)
Azure/azure-sdk-for-java: sdk/spring/README.md

```
[//]: # ({x-version-update-end})

## Spring Boot 3 Support

The source code of Spring Cloud Azure for Spring Boot 3.x can be found on the [feature/spring-boot-3](https://github.com/Azure/azure-sdk-for-java/tree/feature/spring-boot-3) branch.

> #### Disable the JAR signature verification for the native image generation

Reviewer: ```suggestion
#### Spring AOT and Spring native images
```

> You have to disable the JAR signature verification to be able to generate a native image ([see](https://github.com/Azure/azure-sdk-for-java/issues/30320)).

Reviewer: Azure SDK JARs are signed. [Spring Boot 3 does not support today signed JARs](https://github.com/Azure/azure-sdk-for-java/issues/30320) when you run your application with [AOT mode on a JVM](https://docs.spring.io/spring-boot/docs/current/reference/html/deployment.html#deployment.efficient.aot) or you [build an native image](https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html).

> You have to disable the JAR signature verification to be able to generate a native image ([see](https://github.com/Azure/azure-sdk-for-java/issues/30320)).

Reviewer: ```suggestion
Azure SDK JARs are signed. https://github.com/Azure/azure-sdk-for-java/issues/30320 when you run your application with [AOT mode on a JVM](https://docs.spring.io/spring-boot/docs/current/reference/html/deployment.html#deployment.efficient.aot) or you [build an native image](https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html).
```

> You can do this in the following way for GraalVM Native Build Tools:

Reviewer: ```suggestion
You can disable the JAR signature verification in the following way for GraalVM Native Build Tools:
```

> Azure SDK JARs are signed. https://github.com/Azure/azure-sdk-for-java/issues/30320 when you run your application with [AOT mode on a JVM](https://docs.spring.io/spring-boot/docs/current/reference/html/deployment.html#deployment.efficient.aot) or you [build an native image](https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html).

Reviewer: ```suggestion
Azure SDK JARs are signed. [Spring Boot 3 does not support today signed JARs](https://github.com/Azure/azure-sdk-for-java/issues/30320) when you run your application with [AOT mode on a JVM](https://docs.spring.io/spring-boot/docs/current/reference/html/deployment.html#deployment.efficient.aot) or you [build an native image](https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html).
```

> Azure SDK JARs are signed. [Spring Boot 3 does not support today signed JARs](https://github.com/Azure/azure-sdk-for-java/issues/30320) when you run your application with [AOT mode on a JVM](https://docs.spring.io/spring-boot/docs/current/reference/html/deployment.html#deployment.efficient.aot) or you [build an native image](https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html).

Reviewer: ```suggestion
Azure SDK JARs are signed. [Spring Boot 3 does not support today signed JARs](https://github.com/Azure/azure-sdk-for-java/issues/30320) when you run your application with [AOT mode on a JVM](https://docs.spring.io/spring-boot/docs/current/reference/html/deployment.html#deployment.efficient.aot) or you [build a native image](https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html).
```
